views/albums.hbs

```diff
@@ -0,0 +1,5 @@
+<div>
+  {{#each albums}}
+    {{> album}}
```

Comment:
I see you calling this file as your partial, but you don't have anything in that file; since it doesn't have a template to use, your partial won't work.